students/JaNiece/index.css

```diff
@@ -35,7 +35,11 @@ nav a:hover {
   color: white;
 }
 
+<<<<<<< HEAD
```

Comment:
This and the few lines below is a [git merge conflict](https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts) that didn't get properly resolved, so you need to delete the extra lines (38-40 and 42)